keras_cv/models/vgg19.py

```diff
@@ -25,39 +25,52 @@
 
 from keras_cv.models import utils
 
+# Copyright 2022 The KerasCV Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
 
-def VGG19(
-    *,
-    include_rescaling,
-    include_top,
-    classes=None,
-    weights=None,
-    input_shape=(224, 224, 3),
-    input_tensor=None,
-    pooling=None,
-    classifier_activation="softmax",
-    name="VGG19",
-    **kwargs,
-):
-    """Instantiates the VGG19 architecture.
+"""VGG19 model for KerasCV.
+Reference:
+  - [Very Deep Convolutional Networks for Large-Scale Image Recognition](https://arxiv.org/abs/1409.1556) (ICLR 2015)
+"""
 
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import layers
+
+from keras_cv.models import utils
+from keras_cv.models.vgg16 import apply_vgg_block
+
+
+@keras.utils.register_keras_serializable(package="keras_cv.models")
+class VGG19(keras.Model):
+    """Instantiates the VGG19 architecture.
     Reference:
     - [Very Deep Convolutional Networks for Large-Scale Image Recognition](https://arxiv.org/abs/1409.1556) (ICLR 2015)
-
     This function returns a Keras VGG19 model.
```

Comment:
nit: this is no lunger a function, but is now a class. (So the "instantiates" comment above is also not quite correct)

Reply:
Fixed..